modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/DLAdminManagementToolbarDisplayContext.java

```diff
@@ -890,6 +925,12 @@ private long _getFolderId() {
 		return _dlAdminDisplayContext.getFolderId();
 	}
 
+	private String _getLabel(String key, String value) {
+		return StringBundler.concat(
+			LanguageUtil.get(httpServletRequest, key), StringPool.COLON,
+			StringPool.SPACE, value);
+	}
+
```

Comment:
Currently the label is built in several places with this code (yours is also OK):

```java
	String.format(
		"%s: %s",
		LanguageUtil.get(_httpServletRequest,  key),
		HtmlUtil.escape(value));
```

Creating a method for it seems OK. Please refactor all places to use the newly created method. And don't forget to add the call to `HtmlUtil.escape()`.

Reply:
I will use the `HtmlUtil.escape(`, thanks!

I have to think about 
`String.format("%s: %s"` vs `StringBundler.concat(`